test/functional/api/cas/cache.py

```diff
@@ -177,7 +177,7 @@ def set_params_alru(self, alru_params: FlushParametersAlru):
                                               if alru_params.staleness_time else None,
                                               alru_params.flush_max_buffers
                                               if alru_params.flush_max_buffers else None,
-                                              int(alru_params.activity_threshold.total_seconds()
+                                              round(alru_params.activity_threshold.total_seconds()
```

Comment:
I think the core problem here is that `total_seconds()` returns floating point number, so casting result of this function to int would allow to narrow down problem to it's actual root case, plus it prevents potential error accumulation (resulting from the nature of multiplication).